bin/reth/src/dump_stage/hashing_account.rs

```diff
@@ -21,10 +21,14 @@ pub(crate) async fn dump_hashing_account_stage<DB: Database>(
 
     // Import relevant AccountChangeSets
     let tx = db_tool.db.tx()?;
-    let from_transition_rev =
-        tx.get::<tables::BlockTransitionIndex>(from)?.expect("there should be at least one.");
-    let to_transition_rev =
-        tx.get::<tables::BlockTransitionIndex>(to)?.expect("there should be at least one.");
+    let from_transition_rev = tx
+        .get::<tables::BlockMeta>(from)?
+        .expect("there should be at least one.")
+        .transition_at_block();
+    let to_transition_rev = tx
+        .get::<tables::BlockMeta>(to)?
+        .expect("there should be at least one.")
+        .transition_after_block();
```

Comment:
I changed logic here, previously it was `(from,to]` but i changed it to `[from,to]`, having `(from` is always strange

`(` -> not inclusive
`[` -> inclusive